riscv/decode_macros.h

```diff
@@ -42,6 +42,15 @@
   })
 #define WRITE_VSTATUS STATE.log_reg_write[3] = {0, 0};
 
+/* the value parameter needs to be evaluated before writing to the registers */
+#define WRITE_REG_PAIR(reg, value) \
+  if (reg != 0) { \
+    require(reg % 2 == 0); \
+    uint64_t val = value; \
+    WRITE_REG(reg, sext32(val)); \
+    WRITE_REG(reg + 1, (sreg_t(val)) >> 32); \
```

Comment:
Please wrap macro parameters in parens when used in expressions (unless it's otherwise unambiguous, e.g. when separated by the comma operator or by other parens).  For example, writing `if (reg)` would be OK, as would `if ((reg) != 0)`.

Reply:
done in 02fbb0d